model/metrics/gen-ai.yaml

```diff
@@ -43,3 +43,19 @@ groups:
           The `error.type` SHOULD match the error code returned by the Generative AI provider or the client library,
           the canonical name of exception that occurred, or another low-cardinality error identifier.
           Instrumentations SHOULD document the list of errors they report.
+  - id: metric.gen_ai.server.latency.time_per_output_token
+    type: metric
+    metric_name: gen_ai.server.latency.time_per_output_token
+    brief: 'Mean time per output token generated'
+    instrument: histogram
+    unit: "s"
+    stability: experimental
+    extends: metric_attributes.gen_ai
+  - id: metric.gen_ai.server.latency.time_to_first_token
```

Comment:
should we rename the `metric.gen_ai.client.operation.duration` to `metric.gen_ai.client.latency.time_to_last_token`?

Reply:
otherwise it is very inconsistent between client and server

Reply:
`time_to_last_token` isn't the widely used terminology here. It is usually referred to as `request_duration` or `request_latency`. Ideally I would prefer to use `gen_ai.client.request.latency`. `gen_ai.client.request.duration` might be ok too. If `operation.duration` is the Otel standard derived from other metrics, then it might be ok to keep it the same too if that's the recommendation.

For context, today vLLM calls it `request_latency` and TGI and Triton call this `request_duration` - https://docs.google.com/document/d/1SpSp1E6moa4HSrJnS4x3NpLuj88sMXr2tbofKlzTZpk/edit?resourcekey=0-ob5dR-AJxLQ5SvPlA4rdsg&tab=t.0#bookmark=id.bdl7ba5eaec.

For the server metric, I haven't included it in this PR. But I'm planning to call it `gen_ai.server.latency.time_per_request`.

Reply:
Should the names be symmetrical? Like: `gen_ai.client.latency.request_duration` and `gen_ai.server.latency.request_duration`. In future if we need a client's time to first token it will be `gen_ai.client.latency.time_to_first_token`

Reply:
Having the names be symmetrical makes sense. I've created the server metric called `gen_ai.server.latency.request_duration` in this PR. I'll make the client change in a separate PR since it could be considered breaking. 

@lmolkova let me know if you have any objection to changing the client metric name to `gen_ai.client.latency.request_duration` in a separate PR.

Reply:
I'd prefer to keep `gen_ai.client.operation.duration`. Reasons:
- duration implies time-to-last semantics
- there are operations that don't produce tokens - e.g. list models, upload file, other control plane things.
- some operation fail before producing any tokens,
- etc.

It does not prevent us from adding more metrics for time-to-first/time-to-last-tokens.

Note: in case of clients, these metrics would only makes sense for streaming requests.

Reply:
Makes sense. We can keep the client metric the same.